assets/src/edit-story/components/canvas/displayElement.js

```diff
@@ -119,14 +120,24 @@ function DisplayElement({ element, previewMode, page }) {
         previewMode={previewMode}
       >
         <Display element={element} previewMode={previewMode} box={box} />
-        {!previewMode && (
-          <ReplacementContainer hasReplacement={Boolean(replacementElement)}>
-            {replacementElement && (
-              <Replacement element={replacementElement} box={box} />
-            )}
-          </ReplacementContainer>
-        )}
       </WithMask>
+      {!previewMode && (
+        <ReplacementContainer hasReplacement={Boolean(replacementElement)}>
+          {replacementElement && (
+            <WithMask
+              element={replacementElement}
+              fill={true}
+              box={box}
+              style={{
+                opacity: opacity ? opacity / 100 : null,
+              }}
+              previewMode={previewMode}
+            >
+              <Replacement element={replacementElement} box={box} />
+            </WithMask>
+          )}
+        </ReplacementContainer>
+      )}
```

Comment:
So there's  slight issue with this - when you hover over a blob with a different flip setting the dragging image has, the blob can be seen in both states at the same time. This can be fixed by wrapping the "real" display element in a container similar to `ReplacementContainer`, but it just fades "the other way".

@wassgha, WDYT here? I think you made this originally.

Reply:
@barklund Note there's also an existing issue with flipped shapes: https://github.com/google/web-stories-wp/issues/1929

Reply:
I say we punt this to that one. Adding a mention there as well.

Reply:
Or actually in #1111.